Code/Framework/AzCore/AzCore/RTTI/BehaviorContext.h

```diff
@@ -2015,7 +2015,9 @@ namespace AZ
                 }
                 else
                 {
+                    AZ_PUSH_DISABLE_WARNING_GCC(-Wdangling-pointer);
```

Comment:
This statement looks wrong, it should be 
```suggestion
                    AZ_PUSH_DISABLE_WARNING_GCC("-Wdangling-pointer");
```
This will produce an error like:
```
missing option after ‘#pragma GCC diagnostic’ kind [-Werror=pragmas]
```
is that why there is an additional `-Wno-error=pragmas` below?

Reply:
Thanks, fixed that.